src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -700,6 +705,12 @@ namespace PurchasedAirManager {
 
 			SetupOutputVariable( "Zone Ideal Loads Hybrid Ventilation Available Status []", PurchAir( PurchAirNum ).AvailStatus, "System", "Average", PurchAir( PurchAirNum ).Name );
 
+			//air flows
```

Comment:
@mjwitte it looks ok to me. The bracket above the line you commented on is from a much earlier set of lines.
The file uses tabs but all the files use tabs on my machine (is that correct?).
Do you really want me to use spaces instead?